### PR TITLE
feat: add TMYG11

### PR DESCRIPTION
**Pull request by Kouteki**
Add new blog article `The More You Gno 11`

### CODE REVIEW

File: posts/2024-08-30_myg-11/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# The More You Gno 11: Introducing gnobro
+
+As we're gearing up toward the mainnet launch, we haven't forgotten to give some love to minor features that makes everyone's lives easier. This time the spotlight's on **gnobro**.
+
+# Gno Core Updates
+
+## gnobro is Live
+
+What's gnobro, you say? Simply put, it's a terminal based realm browser you can use to explore realms and improve the development experience on 'gnodev'. [See it in action](https://www.php8.ltd/HostLocMJJ/https://github.com/gnolang/gno/pull/2608).
```
Comment (moul) [inline posts/2024-08-30_myg-11/README.md:16]: what is php8.ltd?

File: posts/2024-08-30_myg-11/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - [Handling non-call expression valuedecl values](https://github.com/gnolang/gno/pull/2647)
+    - And more pending reviews!
+- We've added back [coverage support (CodeCov) for txtar tests](https://github.com/gnolang/gno/pull/2377), which make up a majority of our integration testing suite. The txtar tests for the `gnovm` package added an additional 5% coverage. We are currently assessing other packages that suffer from bad txtar coverage.
+- We've added [support for more robust stack traces for Gno-code panics](https://github.com/gnolang/gno/pull/2145), providing a much better UX for the developer. You no longer need to dig through a 5k line log output to figure out what panicked in your Gno code; you'll see the exception stack trace instead.
+- [Variable config command help output is drastically improved](https://github.com/gnolang/gno/pull/2399). In the past you'd need to know exactly what the configuration looks like before modifying or viewing the values. Now these values are conveniently present in the command help output.
+- Last, but not least: let's welcome our new R&D Go Engineer, Antoine. He'll help us scale core components for gno and beyond!
```
Comment (moul) [inline posts/2024-08-30_myg-11/README.md:30]:
```suggestion
- Last, but not least: let's welcome our new R&D Go Engineer, [Antoine](https://github.com/aeddi). He'll help us scale core components for gno and beyond!
```

File: posts/2024-08-30_myg-11/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# Events and Meetups
+
+## Past events
+
+### BUIDL With Cosmos / Web3 Summit
```
Comment (moul) [inline posts/2024-08-30_myg-11/README.md:36]:
```suggestion
### BUIDL With Cosmos / Web3 Summit, Berlin, Germany
```

**Review by moul — APPROVED** [review]

File: posts/2024-08-30_myg-11/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+---
+publication_date: 2024-08-30T00:00:00Z
```
Comment (leohhhn) [inline posts/2024-08-30_myg-11/README.md:2]:
```suggestion
publication_date: 2024-09-02T00:00:00Z
```

**Review by leohhhn — APPROVED** [review]

**Pull request merged**